data-structures/QueueUsingLinkedList.cpp

```diff
@@ -0,0 +1,88 @@
+#include<stdio.h>
+#include <stdlib.h>
+struct node {
```

Comment:
Please use this code style conventions https://google.github.io/styleguide/cppguide.html and use C++ functions and operators like ```new``` and ```delete```